_includes/common/ui/statements-page.md

```diff
@@ -0,0 +1,261 @@
+  statistics -> 'statistics' -> 'cnt' AS execution_count,
+  statistics -> 'statistics' -> 'firstAttemptCnt' AS number_first_attempts,
+  statistics -> 'statistics' -> 'numRows' -> 'mean' AS number_rows_returned_mean,
+  statistics -> 'statistics' -> 'rowsRead' -> 'mean' AS number_rows_read_mean,
+  statistics -> 'statistics' -> 'runLat' -> 'mean' AS runtime_latecy_mean,
+  sampled_plan
+FROM crdb_internal.statement_statistics;
+~~~
+
+## SQL statement fingerprints
+
+The Statements page displays SQL statement fingerprints.
+
+A _statement fingerprint_ represents one or more SQL statements by replacing literal values (e.g., numbers and strings) with underscores (`_`). This can help you quickly identify frequently executed SQL statements and their latencies.
+
+For multiple SQL statements to be represented by a fingerprint, they must be identical aside from their literal values:
+
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (192, 891, 20)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (784, 452, 78)`
+
+The preceding SQL statements have the fingerprint:
+
+`INSERT INTO new_order(product_id, customer_id, no_w_id) VALUES (_, _, _)`
+
+The following statements cannot be represented by the same fingerprint:
+
+- `INSERT INTO orders(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, $3)`
+
+It is possible to see the same fingerprint listed multiple times in the following scenarios:
+
+- Statements with this fingerprint were executed by more than one [`application_name`]({{ link_prefix }}show-vars.html#supported-variables).
+- Statements with this fingerprint were executed both successfully and unsuccessfully.
+
+## Understand the Statements page
+
+Use the Statements page to identify SQL statements that you want to [troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html). This might include statements that are experiencing high latencies, multiple [retries]({{ link_prefix }}transactions.html#transaction-retries), or execution failures. You can optionally create and retrieve [diagnostics](#diagnostics) for these statements.
+
+{{site.data.alerts.callout_success}}
+If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+{{site.data.alerts.end}}
+
+{% if page.cloud == true %}
+<img src="{{ 'images/cockroachcloud/statements_page.png' | relative_url }}" alt="{{ site.data.products.db }} Console Statements Page" style="border:1px solid #eee;max-width:100%" />
+{% endif %}
```

Comment:
I think you should remove or update this screenshot.  It has the old columns and Serverless doesn't support Diagnostics.

<img width="1360" alt="Screen Shot 2021-11-15 at 10 23 51 AM" src="https://user-images.githubusercontent.com/20482960/141834494-39959a13-30ad-43eb-9a07-639ee211c12d.png">

This are the current columns:
<img width="1930" alt="Screen Shot 2021-11-15 at 10 22 49 AM" src="https://user-images.githubusercontent.com/20482960/141834306-3c18b15b-91ee-4ffb-bc03-e26234e4dfef.png">

FWIW, I removed the screenshot from DB Console docs.